lte/gateway/c/oai/tasks/nas5g/.exrc

```diff
@@ -0,0 +1,18 @@
+if &cp | set nocp | endif
```

Comment:
This is a VIM configuration file, which should not be checked into the repo. Please remove it from this PR.

Reply:
.exrc file removed